addons/sale/i18n/sale.pot

```diff
@@ -1491,12 +1491,14 @@ msgstr ""
 #. module: sale
 #. odoo-python
 #: code:addons/sale/models/sale_order_line.py:0
-msgid "Down Payment (Cancelled)"
+#, python-format
+msgid "Down Payment (Canceled)"
```

Comment:
Did you export the translations with the right wizard?
The strings didn't match :eyes:

Reply:
In 17.2, the msgid is with 2 'L's. I should've have also made the change int he `s.o.l` .py file. 
Which is probably the reason why the strings didn't match.